app/models/task.rb

```diff
@@ -20,6 +20,7 @@
 
 class Task < ApplicationRecord
   has_and_belongs_to_many :campaigns
+  has_and_belongs_to_many :events
```

Comment:
That's a good way to put it. I just cringe everytime I read habtm :D but I'll live! =) gonna check the rest and merge.